keyboards/handwired/ortho5x14/keymaps/2u/keymap.c

```diff
@@ -0,0 +1,602 @@
+  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, RESET, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,   XXXXXXX, XXXXXXX, _______, _______,
+  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, AU_ON,  AU_OFF, AG_NORM, AG_SWAP, QWERTY,    XXXXXXX, XXXXXXX, XXXXXXX, _______,
+  XXXXXXX, XXXXXXX, XXXXXXX, MUV_DE,  MUV_IN,  MU_ON,  MU_OFF,   MI_ON,  MI_OFF,  XXXXXXX,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+  XXXXXXX, _______, _______, _______, _______,     _______,        _______,       _______, _______, _______, _______, _______
+)
+,
+
+/* Layer 3 - Mouse Controls
+ * ,-------------------------------------------------------------------------------------------------.
+ * |      |      |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |  F7  |  F8  |  F9  |  F10 |  F11 | F12  |
+ * |------+------+------+------+------+------+------+------+------+------+------+------+------+------|
+ * | PGUP |      |      |MS UP |      |      |SPEED0|      |      |WHUP  |      |      |      |      |
+ * |------+------+------+------+------+------+------+-------------+------+------+------+------+------|
+ * | PGDN |      |MSLeft|MS DN |MSRite|      |SPEED1|      |WHLeft|WHDN  |WHRite|      |      |      |
+ * |------+------+------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |      |      |      |      |      |SPEED2|      | BTN1 | BTN2 | BTN3 |      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Ctrl | Win  | Alt  |Lower |Raise | Space/Enter |     BTN1    | <--  |   v  |   ^  | -->  | Esc  |
+ * `-------------------------------------------------------------------------------------------------'
+ */
+[_MOUSE] = LAYOUT(
+  _______     , XXXXXXX      , KC_F1       , KC_F2          , KC_F3       , KC_F4  , KC_F5        , KC_F6  , KC_F7         , KC_F8        , KC_F9         , KC_F10  , KC_F11  , KC_F12,
+  _______     , XXXXXXX      , XXXXXXX     , KC_MS_UP       , XXXXXXX     , XXXXXXX, KC_MS_ACCEL0 , XXXXXXX, XXXXXXX       , KC_MS_WH_UP  , XXXXXXX       , XXXXXXX , XXXXXXX , XXXXXXX,
+  _______     , XXXXXXX      , KC_MS_LEFT  , KC_MS_DOWN     , KC_MS_RIGHT , XXXXXXX, KC_MS_ACCEL1 , XXXXXXX, KC_MS_WH_LEFT , KC_MS_WH_DOWN, KC_MS_WH_RIGHT, XXXXXXX , XXXXXXX , XXXXXXX,
+  XXXXXXX     , XXXXXXX      , XXXXXXX     , XXXXXXX        , XXXXXXX     , XXXXXXX, KC_MS_ACCEL2 , XXXXXXX, KC_MS_BTN1    , KC_MS_BTN2   , KC_MS_BTN3    , XXXXXXX , XXXXXXX , XXXXXXX,
+  TD(CTL_OSL1), OSM(MOD_LGUI), TD(ALT_OSL1), TD(TD_LayerDn) , KC_L2       ,    TD(TD_SPC_ENT)     ,    KC_MS_BTN1          , KC_LEFT      , KC_DOWN       , KC_UP   , KC_RIGHT, TD(TD_ESC_GRAVE)
+)
+
+};
+
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  #ifdef CONSOLE_ENABLE
+    //uprintf("%s keycode\n", keycode);
+    uprintf("process --> KL: kc: %u, col: %u, row: %u, pressed: %u\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed);
```

Comment:
This change will write the keycode out as a hex value instead of a decimal value, which may be easier to use as most of QMK uses hex. 

```suggestion
    uprintf("process --> KL: kc: 0x%04X, col: %u, row: %u, pressed: %u\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed);
```

Reply:
Done.